react/components/contexts/ripe-context/ripe-context.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { RipeAPI } from "ripe-sdk";
+
+export const RipeContext = React.createContext();
+
+export class RipeProvider extends Component {
+    static get propTypes() {
+        return {
+            options: PropTypes.shape({

```

Comment:
Don't say anything about the structure, otherwise we'll have to replicate the schema of ripe-sdk here.